templates/single-with-sidebar.html

```diff
@@ -1,60 +1,47 @@
-<!-- wp:template-part {"slug":"header","area":"header","tagName":"header"} /-->
+<!-- wp:template-part {"slug":"header","theme":"twentytwentyfour","tagName":"header","area":"header"} /-->
```

Comment:
the theme slug should be gone

Reply:
Ah, that must have been some copy/pasting error. Done.